openfisca_country_template/entities.py

```diff
@@ -42,7 +42,7 @@
 Person = build_entity(
     key = "person",
     plural = "persons",
-    label = u'Person',
+    label = u'A Person represents an individual, the minimal legal entity on which a legislation might be applied.',
```

Comment:
We use "A Person represents", but not "A Household" represents. For consistency and simplicity, I'd remove "A Person represents".